### PR TITLE
scWETH: check Interest rates and update LTV accordingly

### PR DESCRIPTION
**Pull request by dxganta**
Added a script which:

- Queries for the Lido Staking Interest (lr) and AAVE v3 borrow interest (br)
- if br>lr, update ltv to a very low number (almost zero), which is similar to taking no leverage and depositing everything to Lido
- else checks if the ltv is less than the best ltv and updates the ltv if true. This logic will be hit in the scenario where we reduced the ltv to almost zero and then after few days if the lr is greater than the br again.

### CODE REVIEW

File: script/UpdateLtvLeveragedEthMainnet.s.sol
Comment (0xrin1) [inline script/UpdateLtvLeveragedEthMainnet.s.sol:24]: TODO should be addressed before merge

File: test/scWETH.t.sol
Comment (0xrin1) [inline test/scWETH.t.sol:749]: should we give 1e18 and 1e20 a name

could make it easier to understand at a glance

File: test/scWETH.t.sol
Comment (0xrin1) [inline test/scWETH.t.sol:756]: can use lowestLtv var here

**Review by 0xrin1 — CHANGES_REQUESTED** [review]

File: script/UpdateLtvLeveragedEthMainnet.s.sol
Comment (dxganta) [reply script/UpdateLtvLeveragedEthMainnet.s.sol:24]: The main idea is to not merge this PR but port the code here to hardhat in our backend repo.

File: test/scWETH.t.sol
Comment (dxganta) [reply test/scWETH.t.sol]: [Fixed](https://github.com/lindy-labs/sandclock-contracts/pull/99/commits/8a6374d42473bbcc8cc00396a1ce868e23aa6d16)

**Pull request closed without merging**
